### ISSUE TITLE
Instrument Control Implementation

### ISSUE THREAD

**Issue by NickDraper** [mantidproject/mantid#10124]
This issue was originally [TRAC 9281](http://trac.mantidproject.org/mantid/ticket/9281)

This ticket is blocked by : 
 - [ ] [TRAC8925](http://trac.mantidproject.org/mantid/ticket/8925)

Based on the specification of http://trac.mantidproject.org/mantid/ticket/8925

Implement the tools needed within Mantid to make effective use of Instrument Control.

**Comment by NickDraper** [comment]
insufficient detail to justify this issue, a new issue will be created (better) when it is required
